frontend/src/App.tsx

```diff
@@ -21,6 +21,17 @@ axios.interceptors.request.use(function (config) {
 function App() {
     const [userData, setUserData] = useState<UserData[]>([])
 
+    function addUserData(userDataToAdd: UserData) {
+        axios
+            .post("/api/userdata/", userDataToAdd)
+            .then((response) => {
+                setUserData(response.data);
+            })
+            .catch((error) => {
+                console.error("I'm sorry. Something went wrong!" + error);
+                // You can add an error message to display to the user here
```

Comment:
Avoid comments in production code. Possibly not only output the error message as a log, but also make it visible to the user :)